packages/nextcloud/test/client_test.dart

```diff
@@ -29,6 +29,7 @@ void main() {
         final client = NextcloudClient(
           uri,
           httpClient: mockedClient,
+          // ignore: deprecated_member_use_from_same_package
           cookieJar: cookieJar,
```

Comment:
Why not use the interceptor for tests?

Reply:
I'll add tests for the CookieInterceptor but would like to keep the existing test until we remove the deprecated member.